dockers/docker-lldp/start.sh

```diff
@@ -1,5 +1,16 @@
 #!/usr/bin/env bash
 
+# Container image version
+# Acquires build image version as its version during build
+#
+IMAGE_VERSION="0.0.0"
```

Comment:
maybe I am missing something here, if you assign variable here, how would you be able to use the environ variable defined in the docker build?

Reply:
Missed. Just realized and fixed too.